src/ActionModal/utils/networkMessages/terra-testnet.js

```diff
@@ -0,0 +1,9 @@
+import * as CosmosMessages from "./cosmos-hub-mainnet.js"
```

Comment:
should this come from the API?

is `msg.type` the only thing that changed or are the data structures different?

Reply:
I don't know. Maybe there could be structural differences down the line. I didn't even check all Terra transactions yet.